src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -39,3 +48,70 @@ async def exec_aggregation_async(
     ) -> AggregationResult:
         """Execute aggregation nodes"""
         raise NotImplementedError()
+
+
+class APIBasedExecutorProxy(AbstractExecutorProxy):
+    @property
+    def api_endpoint(self) -> str:
```

Comment:
what is this for? add some doc string here?

Reply:
Added.